projects/igniteui-angular/src/lib/navigation-drawer/navigation-drawer.component.html

```diff
@@ -14,7 +14,7 @@
     [class.igx-nav-drawer--disable-animation]="disableAnimation"
     (click)="close()" #overlay>
 </div>
-<aside role="navigation"
+<div
```

Comment:
Shouldn't the correct semantic element be `<nav>`?

Reply:
We already have the nav element inside, that's why I changed it to div, i removed the role since we are using the semantic `<nav>` element, and no role is needed for it.

<img width="1016" alt="image" src="https://github.com/user-attachments/assets/2574fba0-c3bf-4421-b591-40e884d8c845">

Reply:
I think we should leave that for the user to decide, we don't know what will be inside, if we change it to nav this means that the drawer can only be used for navigation purposes, if that's what we want. then OK, I will change it.

Reply:
@simeonoff, @damyanpetev, what is your opinion?

Reply:
@desig9stein I'm saying this because so far, we'd decided by putting `role="navigation"`.

Reply:
@kdinev you are correct, otherwise I will introduce a breaking change.